commands/show.go

```diff
@@ -166,7 +166,7 @@ func showDefaultFormatter(env *Env, snapshot *bug.Snapshot) error {
 		)
 
 		if comment.Message == "" {
-			message = colors.GreyBold("No description provided.")
+			message = colors.BlackBold(colors.White("No description provided."))
```

Comment:
I'm a bit confused, you are setting the foreground color twice. Shouldn't that be one for the bg, one for the fg ?

Reply:
@MichaelMure Good catch, it should be `WhiteBg` instead of `White`. I've fixed it and took the opportunity to squash all review fixes to relevant commits.